org.kde.haruna.yml

```diff
@@ -0,0 +1,181 @@
+---
+app-id: org.kde.haruna
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: haruna
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  - '--device=dri'
+  - '--filesystem=host'
+  - '--talk-name=org.freedesktop.ScreenSaver'
+  - '--own-name=org.mpris.MediaPlayer2.haruna'
+  - '--env=LC_NUMERIC=C'
+modules:
+  - name: haruna
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: 'https://invent.kde.org/multimedia/haruna/-/archive/0.6.3/haruna-0.6.3.tar.gz'
+        sha256: 'c79ec1e351f47faf9a58a6ba7ec3cc05cfdc5423fde0584f2d4081f5058363e3'
```

Comment:
build failure is because it can't find the appstream file.

Reply:
home isn't enough for that? or even xdg-videos?

Reply:
No. For example  have an external drive that  gets mounted in `/run/media`.

Reply:
@hfiguiere ping